local-cli/generator-windows/index.js

```diff
@@ -4,17 +4,112 @@ var chalk = require('chalk');
 var fs = require('fs');
 var path = require('path');
 var yeoman = require('yeoman-generator');
+var utils = require('../generator-utils');
 
 module.exports = yeoman.Base.extend({
   constructor: function () {
     yeoman.Base.apply(this, arguments);
     this.argument('name', { type: String, required: true });
+
+    this.option('ns', {
+      desc: 'Namespace for the application (MyNamespace.MyApp)',
+      type: String,
+      defaults: this.name
+    });
+
+    this.option('upgrade', {
+      desc: 'Specify an upgrade',
+      type: Boolean,
+      defaults: false
+    });
+  },
+
+  configuring: function() {
+    /* TODO: Fix
+    utils.copyAndReplace(
+      this.templatePath('../../../.flowconfig'),
+      this.destinationPath('.flowconfig'),
+      {
+        'Libraries\/react-native\/react-native-interface.js' : 'node_modules/react-native/Libraries/react-native/react-native-interface.js',
+        '^flow/$' : 'node_modules/react-native/flow\nflow/'
+      }
+    );
+    */
+
```

Comment:
I imagine we're supposed to copy some files for Flow to work properly, but we'd have to fix the paths.